src/logic/operate.js

```diff
@@ -0,0 +1,32 @@
+import Big from 'big.js';
+
+const Operate = (numberOne, numberTwo, operation) => {
+  let result = 0;
+  const i = Big(numberOne);
+  const j = Big(numberTwo);
+
+  switch (operation) {
+    case '+':
+      result = i + j;
+      break;
+
+    case '-':
+      result = i - j;
+      break;
+
+    case '*':
+      result = i * j;
+      break;
+
+    case '/':
+      result = i / j;
+      break;
+
+    default:
+      result /= 100;
+      break;
```

Comment:
Job well done but you don't have to do the calculating yourself, that where the use of `big` comes in -> [big js library](https://github.com/favio41/big-js#readme)
Example: case subtraction will be -> `total.minus(next)`

OR

[Mikemcl GitHub](http://mikemcl.github.io/big.js/#mod
)